fastlane/Fastfile

```diff
@@ -238,6 +238,13 @@ platform :ios do
 
   desc "visionOS build"
   lane :build_visionos do |options|
+    pod_lib_lint(
+      verbose: true,
+      podspec:'RevenueCat.podspec', 
+      platforms:"xros",
```

Comment:
Xcode 15.1 beta 3 changes this to `visionos` so this will probably have to change too.